src/apis/doctor/doctor.service.js

```diff
@@ -76,59 +77,49 @@ class DoctorService {
 
     getById(id, callback) {
         return doctorDao.readById(id, (doctorById) => {
-            callback(doctorById);
+            doctorById.dataValues.qualification = [];
```

Comment:
We had decided not to go with this method, right?

Reply:
Rahul  mentioned it should be fine  to go ahead with this, but  suggested it will not be efficient in case of large tables but in our case it is small.May be in the next commit if required i will take help from nilu to write raw query.

Reply:
I don't see the need of it. We don't need the raw query either here. Nilu has written a service which I saw in the noon today and it should work here too. Check with @niluroy and change it accordingly.

Reply:
yeah will do the same here too